File: Programming Basics with J/08. More Exersices/salary.js
function solve (input) {
    let tabs = Number(input[0]);
    let salary = Number(input[1]);

    for( let i =2; i < input.length; i++){
        let website = input[i];

        if( website === "Facebook"){
            salary-=150;
        } else if (website === "Instagram"){
            salary-=100;
        } else if (website === "Reddit"){
            salary -= 50;
        }

        if(salary<=0){
            console.log(`You have lost your salary.`);
            break;
        }
    }

    if(salary>0){
        console.log(`${salary}`);
    }
}



solve(["10", "750", "Facebook", "Dev.bg", "Instagram", "Facebook", "Reddit", "Facebook", "Facebook"])

File: Programming Basics with Java/04. For Loop - Exercise/src/Salary.java
import java.util.Scanner;

public class Salary {
    public static void main(String[] args) {
        Scanner scan = new Scanner (System.in);

        int tabs = Integer.parseInt(scan.nextLine());
        double salary = Integer.parseInt(scan.nextLine());

        for (int i = 0; i < tabs; i++) {
            String currentTab = scan.nextLine();

            if(currentTab.equals("Facebook")){
                salary -= 150;
            } else if(currentTab.equals("Instagram")){
                salary -= 100;
            } else if (currentTab.equals("Reddit")){
                salary -= 50;
            }

            if(salary <= 0){
                break;
            }
        }

        if(salary > 0){
            System.out.printf("%.0f", salary);
        } else {
            System.out.println("You have lost your salary.");
        }
    }
}
